.github/workflows/ios_deploy_staging_to_firebase.yml

```diff
@@ -0,0 +1,55 @@
+name: ios-deploy-to-staging
+on:
+  # Trigger the workflow on push action
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_and_upload_staging_app_to_firebase:
+    name: Build And Upload iOS Staging Application To Firebase
+    runs-on: macOS-latest
+    defaults:
+      run:
+        working-directory: template
```

Comment:
I think this way allows us to build the `template` project without generating it. However, we're working on an EPIC to revamp the generating script 👉  The `template` project will not be the normal Flutter project and we can not build it without generating.

Please remove this config and add the steps for project generating.